### PR TITLE
update doc with SME feedback

### PR DESCRIPTION
**Pull request by cbudz**
Hi Dayle,

Here's the updates to the operational alerts table. Once they're approved and merged, I'll backport the content to darga/euwe/fine. 

-Chris

### CODE REVIEW

File: doc-Monitoring_Alerts_and_Reporting/topics/Operational_Alerts.adoc [hunk 1]
```diff
@@ -51,7 +51,7 @@
 					
 | 
 						
-							Alert is raised when one EVM server detects that another EVM Server has not responded in (2 minutes).
+							Alert is raised when one server detects that another server has not responded in (2 minutes).
```
Comment (dayleparker) [inline doc-Monitoring_Alerts_and_Reporting/topics/Operational_Alerts.adoc]: I think the brackets around (2 minutes) are probably best to be removed here. What do you think?

File: doc-Monitoring_Alerts_and_Reporting/topics/Operational_Alerts.adoc [hunk 2]
```diff
@@ -158,7 +158,7 @@
 					
 | 
 						
-							Something is filling the disk such as temp files used by the operating system such as, yum updates and normal /tmp files, or EVM temp files in /var/lib/data/miqtemp/.
+							Something is filling the disk such as temp files used by the operating system such as, yum updates and normal /tmp files, or temp files in /var/lib/data/miqtemp/.
```
Comment (dayleparker) [inline doc-Monitoring_Alerts_and_Reporting/topics/Operational_Alerts.adoc]: Very small thing, but the comma after the second "such as" should be removed -- or maybe could the sentence be reworded so there are not two "such as" instances in it?

File: doc-Monitoring_Alerts_and_Reporting/topics/Operational_Alerts.adoc [hunk 3]
```diff
@@ -297,7 +297,7 @@
 						
 					
 |
-
+							An influx of events from the Virtual Center or RHEV causes an inability of EVM/CFME to handle the capacity at which they are being queued. Utilize the Event Handler Configuration to filter events that are causing problematic queue table growth.
```
Comment (dayleparker) [inline doc-Monitoring_Alerts_and_Reporting/topics/Operational_Alerts.adoc]: We generally shouldn't abbreviate product names (though maybe this is different in tables)... however if we are OK to keep the abbreviation for "RHEV", it should be corrected to "RHV".

**Comment by cbudz** [comment]
Hi Dayle,

I've made changes based on your feedback. I did replace RHEV with the full product title, and changed that awkward sentence.  Let me know what you think.

-Chris

**Comment by dayleparker** [comment]
Hi Chris,
Everything looks excellent -- thanks for making those edits!
Cheers,
Dayle

**Pull request merged**